### ISSUE TITLE
Investigate dealbot data propagation to Pando

### ISSUE THREAD

**Issue by davidd8** [filecoin-project/dealbot#388]
Investigate reports that dealbot data is not propagating to the Pando service.

**Comment by masih** [comment]
Looking at the logs [no publication failure](https://app.logz.io/#/dashboard/kibana/discover/4aa58eb0-8f12-11ec-998e-5789b5b42109?_a=(columns:!(message),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'logzioCustomerIndex*',key:cluster,negate:!f,params:(query:mainnet-us-east-1-eks),type:phrase),query:(match_phrase:(cluster:mainnet-us-east-1-eks))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'logzioCustomerIndex*',key:kubernetes.namespace,negate:!f,params:(query:ntwk-mainnet-dealbot),type:phrase),query:(match_phrase:(kubernetes.namespace:ntwk-mainnet-dealbot))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'logzioCustomerIndex*',key:kubernetes.container.name,negate:!f,params:(query:dealbot-controller),type:phrase),query:(match_phrase:(kubernetes.container.name:dealbot-controller))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'logzioCustomerIndex*',key:logger,negate:!f,params:(query:controller),type:phrase),query:(match_phrase:(logger:controller)))),index:'logzioCustomerIndex*',interval:auto,query:(language:lucene,query:'%22Failed%20to%20publish%22'),sort:!(!('@timestamp',desc)))&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-30d,to:now))&discoverTab=logz-logs-tab&accountIds=true&switchToAccountId=104138) is occurring. 

It seems no data is being produced at all to be published in the first place. 

Can you point me to logs that suggest otherwise please?

**Comment by willscott** [comment]
Right. the dealbot model is that data is only propagated to pando when a dealbot daemon is 'finalized' / rotated. This is so that the active IPs / identities of daemons still doing in-process work are not exposed publicly until that bot has been swapped out.

We'll need to continue understanding usage by data programs, which has stopped rotated daemon processes regularly unless we ask them to because disks fill up.
